src/bin/teleport.rs

```diff
@@ -30,7 +31,7 @@ use teleport::{
 #[command(next_line_help = true)]
 struct ArgsWithWalletFile {
     /// Wallet file Name
-    #[arg(long, short, default_value = "wallet.teleport", value_parser = clap::value_parser!(PathBuf))]
+    #[arg(long, short, default_value = get_default_wallet_path().into_os_string(), value_parser = clap::value_parser!(PathBuf))]
```

Comment:
I won't even bother fixing anything in the bin module. Just make it pass compilation.  This app isn't much useful for anything. All the high level operations should be done via the `integration` tests at this stage. The library isn't in a demo-able state and we will rewrite the binaries completely later on anyway.